src/v2/index.ejs

```diff
@@ -5,6 +5,17 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no" />
   <meta charset='utf-8' />
 
+  <!-- onetrust cookie consent, must be the first js loaded -->
+  <%% if ( !sd.THIRD_PARTIES_DISABLED && sd.ONETRUST_SCRIPT_ID ) { %>
```

Comment:
@icirellik can chime in on the necessity of this, but we've been adding all 3rd party scripts to this additional `disable` object so that we can toggle off and on specifically: https://github.com/artsy/force/blob/master/src/v2/server.ts#L97

Reply:
oh. what's the use case? is it just for local dev?

Reply:
I _think_ this is lighthouse related, for narrowing down on script that impact performance.

Reply:
i see. thanks. @icirellik - i will add if helps.

Reply:
Yup, it was added as a way to disable locally and possibly in review apps.

Reply:
There are ways to disable scripts. `THIRD_PARTIES_DISABLED` is meant to disable all 3rd party scripts for testing locally and possibly in a review app.  What @damassi is pointing out is used to disable individual 3rd party scripts using a URL parameter that is Calibre uses.

Reply:
thanks. added to `disable` object.